packages/integration-test/src/utils.js

```diff
@@ -0,0 +1,43 @@
+const { note } = require('aztec.js');
+
+const utils = {};
+
+/**
+ * Make the first letter of a given single word input string capitalised
+ *
+ * @capitaliseFirstChar
+ * @param {string} stringToCapitalise - input for which the first letter should be capitalised
+ *
+ * @returns {string} - input string with first letter capitalised
+ */
+utils.capitaliseFirstChar = (stringToCapitalise) => {
+    return stringToCapitalise.charAt(0).toUpperCase() + stringToCapitalise.slice(1);
```

Comment:
More informational: Strings are arrays of chars. You can access a char by just using square brackets.

Reply:
Didn't realise, nice